solidity/random-beacon/tasks/initialize.ts

```diff
@@ -1,14 +1,23 @@
-/* eslint-disable no-console */
 import { task, types } from "hardhat/config"
 
-import type { BigNumberish, BigNumber } from "ethers"
+import {
+  calculateTokensNeededForStake,
+  mint,
+  stake,
+  authorize,
+  register,
+} from "./utils"
+
+import type { BigNumberish } from "ethers"
 import type { HardhatRuntimeEnvironment } from "hardhat/types"
 
-const TASK_INITIALIZE = "initialize"
-const TASK_MINT = `${TASK_INITIALIZE}:mint`
-const TASK_STAKE = `${TASK_INITIALIZE}:stake`
-const TASK_AUTHORIZE = `${TASK_INITIALIZE}:authorize`
-const TASK_REGISTER = `${TASK_INITIALIZE}:register`
+export const TASK_INITIALIZE = "initialize"
+export const TASK_MINT = `${TASK_INITIALIZE}:mint`
+export const TASK_STAKE = `${TASK_INITIALIZE}:stake`
+export const TASK_AUTHORIZE = `${TASK_INITIALIZE}:authorize`
+export const TASK_REGISTER = `${TASK_INITIALIZE}:register`
+const TASK_AUTHORIZE_BEACON = `${TASK_AUTHORIZE}:beacon`
```

Comment:
I think this requires some explanation in the comments. By looking at the beacon code it tells you nothing that half of these tasks are used by `ecdsa` and task like `TASK_AUTHORIZE_BEACON` might appear confusing, ie why not use just `TASK_AUTHORIZE`

Reply:
d2d851d87f113c9d1796d05787735fb65634dc56